src/query/contact/contact_support_map_support_map.rs

```diff
@@ -19,12 +19,16 @@ where
 {
     let simplex = &mut VoronoiSimplex::new();
     match contact_support_map_support_map_with_params(pos12, g1, g2, prediction, simplex, None) {
-        GJKResult::ClosestPoints(point1, point2_1, normal1) => {
+        GJKResult::ClosestPoints(point1, point2_1, Ok(normal1)) => {
             let dist = (point2_1 - point1).dot(&normal1);
             let point2 = pos12.inverse_transform_point(&point2_1);
             let normal2 = pos12.inverse_transform_unit_vector(&-normal1);
             Some(Contact::new(point1, point2, normal1, normal2, dist))
         }
+        GJKResult::ClosestPoints(_, _, Err(_)) => {
+            // TODO: propagate the error.
+            None
+        }
```

Comment:
this is a breaking change, which would end up in breaking access succeeding before (but probably behaving slightly incorrectly).

Reply:
I consider support map without normal information incorrect

Reply:
TODO: add warn! + test example 3d convex decomposition